### PR TITLE
LFVM: cosmetic code changes

### PR DESCRIPTION
**Pull request by simonlechner**
While I was catching up on all the changes from the refactoring, I spotted a few cosmetic details. This PR consists of renaming, formating and other changes which do not alter any functionality.

### CODE REVIEW

File: go/interpreter/lfvm/converter.go [hunk 1]
```diff
@@ -174,120 +174,8 @@ func convertWithObserver(
...
-			}
-			if op0 == vm.DUP2 && op1 == vm.MSTORE {
-				res.appendCode(DUP2_MSTORE)
-				return 1
-			}
-			if op0 == vm.DUP2 && op1 == vm.LT {
-				res.appendCode(DUP2_LT)
-				return 1
-			}
+		if n := appendSuperInstructions(res, pos, code); n > 0 {
```
Comment (LuisPH3) [inline go/interpreter/lfvm/converter.go:177]: I had this one on my list :+1:

**Review by LuisPH3 — APPROVED** [review]

File: go/interpreter/lfvm/opcode.go [hunk 2]
```diff
@@ -285,6 +285,8 @@ const (
 )
 
 var toString = map[OpCode]string{
+	DATA:    "DATA",
```
Comment (facuMH) [inline go/interpreter/lfvm/opcode.go:288]: why move these two up here ?

**Review by facuMH — APPROVED** [review]
thanks for these !! eventually someone would have had to do it

File: go/interpreter/lfvm/opcode.go (hunk 2 above)
Comment (simonlechner) [reply go/interpreter/lfvm/opcode.go:288]: To have all the LFVM-specific instructions gathered in the beginning, followed by the super instructions.

**Review by HerbertJordan — APPROVED** [review]
Thanks for cleaning up those code style issues.

**Pull request merged**